### ISSUE TITLE
Excessive log spam for 'Creating Creative Mode Tab' message

### ISSUE THREAD

**Issue by rshchekotov** [Hexeption/AEInfinityBooster#16]
Good day,

I hope this finds you well. While working on creating a modpack and debugging some errors, I stumbled upon a repetitive log message related to AEInf.Booster. Specifically, the message:
```
[Render thread/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
```
is logged 77 times in the console. I traced it back to [this line](https://github.com/Hexeption/AEInfinityBooster/blob/5a329335f1c02f40b27a122566fd136c2124ee7b/src/main/java/uk/co/hexeption/aeinfinitybooster/AEInfinityBooster.java#L38) in the repository.

I would assume this also scales with the mod count currently I'm working with just 32 mods, but there are far bigger modpacks.

Would you consider decreasing the log level to debug for this? My thought process is that this seems more like an internal message rather than something an end-user would need to see frequently. If users are interested, they can always adjust their logger settings to access debug logs.

Another suggestion would be to add the actual creative-tab id, provided it is possible, to the debug-message so that one may identify what exactly happens.

Thank you for your time and consideration.

Warm regards,
Roman

**Comment by Zelioth** [comment]
Hey, can also confirm could you please fix this. 

FO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:08] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:09] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:09] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:09] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:09] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:09] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:09] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:09] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:09] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:09] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:09] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab
[18.11 21:41:09] [Server] [main/INFO] [uk.co.he.ae.AEInfinityBooster/]: Creating Creative Mode Tab

**Comment by SirEndii** [comment]
Fixed in #19
